### PR TITLE
Fixed the project template and project type

### PR DESCRIPTION
**Pull request by EranGon**

### CODE REVIEW

File: src/SDK/SmartDetectors.ProjectTemplate/EmptyAlert.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿namespace $safeprojectname$
 {
     using Microsoft.Azure.Monitoring.SmartDetectors;
+    using Microsoft.Azure.Monitoring.SmartDetectors.AlertPresentation;
```
Comment (bermanguy) [inline src/SDK/SmartDetectors.ProjectTemplate/EmptyAlert.cs]: not necessary

File: src/SDK/SmartDetectors.ProjectTemplate/LogSearchAlert.cs [hunk 2]
```diff
@@ -23,20 +25,20 @@ public class $alertName$ : Alert
...
 
         /// <summary>
         /// Gets or sets the number of records
         /// </summary>
-        [AlertPresentationProperty(AlertPresentationSection.Property, "Number of records", Order = 2, InfoBalloon = "The number of records")]
+        [TextProperty("Number of records", Order = 2)]
         public int Count { get; set; }
 
         /// <summary>
         /// Gets the query that displays the number of records over time
```
Comment (bermanguy) [inline src/SDK/SmartDetectors.ProjectTemplate/LogSearchAlert.cs]: or sets

File: src/SDK/SmartDetectors.ProjectTemplate/LogSearchAlert.cs [hunk 3]
```diff
@@ -23,20 +25,20 @@ public class $alertName$ : Alert
...
         /// </summary>
-        [AlertPresentationProperty(AlertPresentationSection.Chart, "Number of records over time for {ResourceName}", Order = 3, InfoBalloon = "The number of records over time")]
-        public string CountChart => $@"$query$";
+        [ChartProperty("Number of records over time for {ResourceName}", ChartType.LineChart, ChartAxisType.DateAxis, ChartAxisType.NumberAxis, Order = 3)]
+        public List<ChartPoint> CountChart { get; set; }
```
Comment (bermanguy) [inline src/SDK/SmartDetectors.ProjectTemplate/LogSearchAlert.cs]: maybe be a real gever and add a table property as well

File: src/SDK/SmartDetectors.ProjectTemplate/EmptyAlert.cs (hunk 1 above)
Comment (EranGon) [reply src/SDK/SmartDetectors.ProjectTemplate/EmptyAlert.cs]: i'd rather leave it, to save the developer the import

---
In reply to: [280450306](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/62#discussion_r280450306) [](ancestors = 280450306)

File: src/SDK/SmartDetectors.ProjectTemplate/LogSearchAlert.cs (hunk 3 above)
Comment (EranGon) [reply src/SDK/SmartDetectors.ProjectTemplate/LogSearchAlert.cs]: not for this change - it's the bare minimum to make it work

---
In reply to: [280451874](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/62#discussion_r280451874) [](ancestors = 280451874)

**Pull request merged**